luaui/Widgets/gui_top_bar.lua

```diff
@@ -655,7 +668,18 @@ local function updateWind()
 	end)
 
 	if WG['tooltip'] ~= nil then
-		WG['tooltip'].AddTooltip('wind', area, Spring.I18N('ui.topbar.wind.tooltip', { titleColor = textTitleColor, warnColor = textWarnColor }))
+		risk=riskWind[minWind]
+		if risk ~= nil then
+			risk=risk[maxWind]
+		end
+		if risk == nil then
+			if minWind+maxWind >= 0.5 then
```

Comment:
Where does the 0.5 come from?

Reply:
Other places in gui_top_bar.lua use "if minWind+maxWind >= 0.5 then" to special case no-wind maps. So I'm personally not sure why that method is used.